### PR TITLE
nodejs json writer updates, and #2210 fix

### PR DESCRIPTION
**Pull request by universalmind303**
closes https://github.com/pola-rs/polars/issues/2210

### CODE REVIEW

File: polars/polars-arrow/Cargo.toml [hunk 1]
```diff
@@ -9,7 +9,7 @@ description = "Arrow interfaces for Polars DataFrame library"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-arrow = { package = "arrow2", git = "https://github.com/jorgecarleitao/arrow2", rev = "d14ae86c69cd76957adec3b14bb62d93732b43c9", default-features = false }
+arrow = { package = "arrow2", git = "https://github.com/jorgecarleitao/arrow2", rev = "25a35f87761bd364860264afbba56d33b3ce74d1", default-features = false }
```
Comment (universalmind303) [inline polars/polars-arrow/Cargo.toml:12]: this version update is the only change I made outside of `nodejs-polars/*`

File: polars/polars-core/Cargo.toml [hunk 2]
```diff
@@ -158,7 +158,7 @@ unsafe_unwrap = "^0.1.0"
 package = "arrow2"
 git = "https://github.com/jorgecarleitao/arrow2"
 # git = "https://github.com/ritchie46/arrow2"
-rev = "d14ae86c69cd76957adec3b14bb62d93732b43c9"
+rev = "25a35f87761bd364860264afbba56d33b3ce74d1"
```
Comment (universalmind303) [inline polars/polars-core/Cargo.toml:161]: here too

File: polars/polars-io/Cargo.toml [hunk 3]
```diff
@@ -31,7 +31,7 @@ private = []
 [dependencies]
 ahash = "0.7"
 anyhow = "1.0"
-arrow = { package = "arrow2", git = "https://github.com/jorgecarleitao/arrow2", rev = "d14ae86c69cd76957adec3b14bb62d93732b43c9", default-features = false }
+arrow = { package = "arrow2", git = "https://github.com/jorgecarleitao/arrow2", rev = "25a35f87761bd364860264afbba56d33b3ce74d1", default-features = false }
```
Comment (universalmind303) [inline polars/polars-io/Cargo.toml:34]: and here

**Comment by ritchie46** [comment]
Thanks!

**Pull request merged**